src/pages/Admin/Venue/Details/VenueDetails.tsx

```diff
@@ -44,13 +40,7 @@ type EditRoomType = RoomData_v2 & {
 const noop = () => {};
 
 const VenueDetails: React.FC<VenueDetailsProps> = ({ venue }) => {
-  const {
-    name,
-    owners,
-    id: venueId,
-    rooms,
-    mapBackgroundImageUrl = DEFAULT_MAP_BACKGROUND,
-  } = venue;
+  const { name, owners, id: venueId, rooms, mapBackgroundImageUrl } = venue;
```

Comment:
Since you removed the default on `mapBackgroundImageUrl`, does anything else here need to change now since it may no longer be defined/provided?

Reply:
Nope, if it's undefined or not selected the choose background options will appear and let you choose one.

Reply:
Sounds good.